packages/shell-api/src/shell-log.ts

```diff
@@ -0,0 +1,68 @@
+import type ShellInstanceState from './shell-instance-state';
+import { shellApiClassDefault, ShellApiClass } from './decorators';
+
+const instanceStateSymbol = Symbol.for('@@mongosh.instanceState');
+
+/**
+ * This class contains the *global log* property that is considered part of the immediate shell API.
+ */
+@shellApiClassDefault
+export default class ShellLog extends ShellApiClass {
+  // Use symbols to make sure these are *not* among the things copied over into
+  // the global scope.
+  [instanceStateSymbol]: ShellInstanceState;
+  log: {
+    info: (message: string, attr?: unknown) => void;
+    warn: (message: string, attr?: unknown) => void;
+    error: (message: string, attr?: unknown) => void;
+    fatal: (message: string, attr?: unknown) => void;
+    debug: (message: string, attr?: unknown, level?: 1 | 2 | 3 | 4 | 5) => void;
+  };
```

Comment:
This is surprising – I'd have expected `ShellLog` to be a class with `info`, `warn`, etc. methods. Right now, this class doesn't really do anything on its own, right?